imports/ui/components/setup/dropdown/PrevEduDropdown.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Select from 'react-select';
+
+/*
+Temporary course option.
+@todo: Transfer Course array into another admin database
+*/
+const Edu =[
```

Comment:
We probably don't need to worry about the hard-coded values here since JC, Poly and Others are quite general already. But take note! @bsweeyee @IamVacer